Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -1201,11 +1201,10 @@ WI.hideSplitConsole = function()
 
 WI.showConsoleTab = function(requestedScope, options = {})
 {
-    requestedScope = requestedScope || WI.LogContentView.Scopes.All;
-
     WI.hideSplitConsole();
 
-    WI.consoleContentView.scopeBar.item(requestedScope).selected = true;
+    if (requestedScope)
```

Comment:
NIT: if you really wanted to be ideal we'd move `requestedScope` to be inside `options` so that callers don't have do provide it if they don't need it (e.g. no need to have a `const requestedScope = null;`)
```js
if (options.requestedScope)
    WI.consoleContentView.scopeBar.item(options.requestedScope).selected = true;
```

Reply:
> While we're at this though, since the Console tab technically supports enabling multiple levels at once (e.g. Errors and Warnings), should I go ahead and change `requestedScope` to `requestedScopes` and make it take an array of scopes instead? Or do you think that'd be too many tangential changes to this commit was meant to do?
> 
> CC @rcaliman-apple

The only things that currently trigger the codepath to show Console with a specific scope are the errors and warnings buttons in the Tab bar:

https://github.com/WebKit/WebKit/blob/1c84cd6c970271eabe238a7e4eff9adced5614ce/Source/WebInspectorUI/UserInterface/Base/Main.js#L455-L468

Since there's no caller with multiple scopes, I don't think it's necessary to convert `requestedScope` to an array now. We can do it whenever it's needed.